### DISCUSSION TITLE
Support for chat from other streaming services

### DISCUSSION THREAD

**Discussion by Hyperkind** [Chatterino/chatterino2#4918] [Ideas]
With the recent update from Twitch to allow multi-streaming to other services, is there plans to allow chat from other streaming services to be enabled with Chatterino?

**Comment by Felanbird** [comment]
There are no plans to support any services other than Twitch, if another service comes along that actually uses IRC, it's already possible to connect to that with chatterino.

#1177 #2162 #3140 #3298 #4676 #4707
